<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionsDialog</class>
 <widget class="QDialog" name="ConnectionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connections</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="moveUp">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="editor.qrc">
         <normaloff>:/images/16x16/go-up.png</normaloff>:/images/16x16/go-up.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="moveDown">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="editor.qrc">
         <normaloff>:/images/16x16/go-down.png</normaloff>:/images/16x16/go-down.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="remove">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="editor.qrc">
         <normaloff>:/images/16x16/edit-delete.png</normaloff>:/images/16x16/edit-delete.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="NodeConnectionsList" name="connectionsTable"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NodeConnectionsList</class>
   <extends>QTreeView</extends>
   <header>nodeconnectionslist.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="editor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
